motion/thruster_interface_auv/package.xml

```diff
@@ -7,14 +7,12 @@
   <maintainer email="vortex.git@vortexntnu.no">vortex</maintainer>
```

Comment:
Also, you can add your email and name here and maybe update the description of the package.